api/src/main/resources/cb_search_api.yaml

```diff
@@ -296,56 +296,88 @@ definitions:
       - hasAttribute
     properties:
       id:
-        description: id of the criteria
+        description: >
+          Primary identifier which is unique within a CDR version. Value may
+          not be stable across different CDR versions.
         type: integer
         format: int64
       parentId:
-        description: parent id of the criteria
+        description: >
+          The parent id of the criteria. 0 if this is the root node of a
+          criteria tree.
         type: integer
         format: int64
       type:
-        description: type of criteria
+        description: The tree type of this criteria, see TreeType.
+        # TODO: Use TreeType enum?
         type: string
       subtype:
-        description: subtype of this criteria
+        description: The tree subtype of this criteria, see TreeSubType.
+        # TODO: Use TreeSubType enum?
         type: string
       code:
+        # TODO: Some of the criteria table has null codes, but it's required
+        # here. What does that mean and how does that manifest in the API?
         description: >
-          code that identifies this criteria, ex: icd9, icd10 or cpt all
-          have codes that map to the concept_code column in the concept
-          table.
+          Code that identifies this criteria. In some vocabularies such as ICD9
+          and ICD10, this code captures the tree hierarchy, e.g. '001.002.003'.
+          Multiple criteria may exist for the same code within a CDR version if
+          a given concept has multiple entries at different locations in the
+          criteria tree (this is common in SNOMED).
+
+          Criteria codes should generally be stable across CDR versions.
```

Comment:
@jscherdin do you mind double-checking my statements here about IDs and how we expect them to be affected across CDR version changes?

In summary what I've documented here:
id - not stable across CDR verisons
code - "mostly"? stable across CDR versions
conceptIds - also "mostly" stable?

Reply:
ID is not stable. Code and concept ID - yes mostly stable is a good way to put it